src/routes/CreateSafePage/components/SafeCreationProcess.tsx

```diff
@@ -136,6 +137,12 @@ const createNewSafe = (userAddress: string, onHash: (hash: string) => void): Pro
         // Monitor the latest block to find a potential speed-up tx
         txMonitor({ sender: userAddress, hash: txHash, data: deploymentTx.encodeABI() })
           .then((txReceipt) => {
+            // txMonitor returns the txReceipt from `getTransactionReceipt` which doesn't
```

Comment:
My parser broke her. Which doesn't throw you mean?

Reply:
Yep. Nice catch!